compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -30,12 +30,11 @@ use tracing::debug;
 pub fn finalize<'ll, 'tcx>(cx: &CodegenCx<'ll, 'tcx>) {
     let tcx = cx.tcx;
 
-    // Ensure LLVM supports Coverage Map Version 4 (encoded as a zero-based value: 3).
-    // If not, the LLVM Version must be less than 11.
-    let version = coverageinfo::mapping_version();
-    if version != 3 {
-        tcx.sess.fatal("rustc option `-Z instrument-coverage` requires LLVM 11 or higher.");
-    }
+    // While our bundled LLVM might support Coverage Map Version 6
+    // (encoded as a zero-based value: 5), we clamp that to Version 5,
+    // as Version 6 would require us to use the 0-th filename as a path prefix
+    // for all other relative paths, which we don't take advantage of right now.
+    let version = coverageinfo::mapping_version().max(4);
```

Comment:
I have no idea if this can be related to the CI failure, but the comment seems to not match the behavior of the code : if mapping_version() is 5 (Coverage Map Version 6), `5.max(4)` will return 5.

Reply:
Oh, you are absolutely right! I will fix this in a minute with another rebase.